snapfaas/src/resource_manager.rs

```diff
@@ -160,10 +164,62 @@ impl ResourceManager {
                             Err(Error::InsufficientEvict)
                         }
                     }
-                    // Just return all other errors
                     _ => Err(e)
                 }
             })
+
```

Comment:
Is the remote scheduler using this file? If yes, can we just delete these commented lines (All histories are kept anyway). And for the purpose of organization, can we move this to the `sched/` folder?